content/en/docs/concepts/services-networking/service.md

```diff
@@ -1168,12 +1069,67 @@ HTTP requests will have a `Host:` header that the origin server does not recogni
 TLS servers will not be able to provide a certificate matching the hostname that the client connected to.
 {{< /warning >}}
 
-{{< note >}}
-This section is indebted to the [Kubernetes Tips - Part
-1](https://akomljen.com/kubernetes-tips-part-1/) blog post from [Alen Komljen](https://akomljen.com/).
-{{< /note >}}
```

Comment:
Dropping this mention after ≅ 5 years; the page has changed quite a bit since then.